### PR TITLE
Support more <ol> list styles

### PR DESCRIPTION
**Pull request by TobyAsE**
I had already done a few `list-style-type`s so I thought why not add a few more?
This adds support for  `list-style-type:` `upper-latin`, `upper-alpha`, `lower-latin`, `lower-alpha` and `decimal-leading-zero`.

The conversion number->alphabet is a bit tricky since the alphabet doesn't have a zero. In case anybody else finds this, the term you want to search for is "bijective numeration" and "bijective base".
Also I made it very alphabet-agnostic to support other easy counting-methods using different charsets. There are a lot out there.
 
I noticed that the markers can collide with the list item itself when they become wider. This needs to be tackled in another PR (issue: #6567) as it seems to not be that trivial.

### CODE REVIEW

File: Userland/Libraries/LibWeb/Layout/ListItemMarkerBox.cpp [hunk 1]
```diff
@@ -21,6 +21,21 @@ ListItemMarkerBox::~ListItemMarkerBox()
...
+        number--;
+        auto remainder = number % base;
+        number -= remainder;
+        output_string.append(alphabet[remainder]);
+        number /= base;
+    };
+    output_string.append(alphabet[number - 1]);
+    return output_string.to_string().reverse();
+}
+
```
Comment (alimpfard) [inline Userland/Libraries/LibWeb/Layout/ListItemMarkerBox.cpp:42]: This also exists in Spreadsheet (`String Spreadsheet::convert_to_string(size_t value, unsigned base = 26, StringView map = {})`) along with its inverse, maybe extract that and use it here?

Comment (TobyAsE) [reply Userland/Libraries/LibWeb/Layout/ListItemMarkerBox.cpp:42]: Ah shoot! Let's take this then. Should we move the functionality to String and friends somewhere?

Comment (alimpfard) [reply Userland/Libraries/LibWeb/Layout/ListItemMarkerBox.cpp:42]: Yeah, probably `String::bijective_base_from(number, <base>, map)` or something?

Comment (TobyAsE) [reply Userland/Libraries/LibWeb/Layout/ListItemMarkerBox.cpp:42]: Sounds good.
I will do that ;)

Comment (awesomekling) [reply Userland/Libraries/LibWeb/Layout/ListItemMarkerBox.cpp:42]: I'll merge this as-is, and we can follow up with changes to add a string helper for bijective basin'.

File: Userland/Libraries/LibWeb/CSS/Identifiers.json [hunk 2]
```diff
@@ -171,4 +173,4 @@
   "xxx-large",
   "zoom-in",
   "zoom-out"
-]
\ No newline at end of file
+]
```
Comment (awesomekling) [inline Userland/Libraries/LibWeb/CSS/Identifiers.json]: Please make sure all commit messages have a `Category:` prefix.

Also, please configure your editor to add a newline at the end of files.

**Review by awesomekling — CHANGES_REQUESTED** [review]

File: Userland/Libraries/LibWeb/CSS/Identifiers.json (hunk 2 above)
Comment (TobyAsE) [reply Userland/Libraries/LibWeb/CSS/Identifiers.json]: Somehow clang-format removed the trailing newline in my setup. It's fixed now.

**Pull request merged**